[Bug Report] missing types PwaInjection with nuxt

```bash
.nuxt/templates.pwa.client.74d38fec.ts:5:35 - error TS2307: Cannot find module '../src/types' or its corresponding type declarations.

5 import { type PwaInjection } from '../src/types'
                                    ~~~~~~~~~~~~~~


Found 1 error in .nuxt/templates.pwa.client.74d38fec.ts:5
```

### How to reproduce

```bash
git clone https://github.com/antfu/vitesse-nuxt3
pnpm i
npm run typecheck
```

--- a/templates/pwa.client.ts
+++ b/templates/pwa.client.ts
@@ -2,7 +2,7 @@ import { ref, reactive, nextTick, type UnwrapNestedRefs } from 'vue'
 import { useRegisterSW } from 'virtual:pwa-register/vue'
 import { defineNuxtPlugin } from '#imports'
 
-import { type PwaInjection } from '../src/types'
+import { type PwaInjection } from '@vite-pwa/nuxt'
 
 // @ts-ignore
 const options: { periodicSyncForUpdates: number; installPrompt?: string } = <%= JSON.stringify(options) %>

